src/2_RTB/AnalisiDeiRequisiti/content.typ

```diff
@@ -0,0 +1,459 @@
+
+#set list(marker: ([•], [--]))
+#set enum(numbering: "1.a.")
+
+= Introduzione
+
+== Scopo del documento
+Il presente documento ha lo scopo di fornire una descrizione dettagliata dei casi d'uso e dei requisiti del progetto _InnovaCity_. Questi risultano dall'analisi del capitolato C6 presentato dalla Proponente _Sync_ Lab e dalla successiva interazione diretta con essa attraverso gli incontri svolti.
+
+== Scopo del prodotto
+Lo scopo del prodotto è la realizzazione di una piattaforma di monitoraggio, che metta insieme i dati prodotti da dispositivi (IoT e non) e sensori di una città. Questa piattaforma permetterà all'#glossary("amministratore pubblico") di ottenere una visione d'insieme della condizione della città e, in base a questo, prendere decisioni informate e tempestive sulla gestione delle risorse e sull'implementazione di servizi. I dati visualizzati da questa piattaforma dovranno venire prodotti con una simulazione randomica che sia verosimile. In alternativa, potranno essere utilizzati dati reali accessibili liberamente.
+
+== Riferimenti
+
+=== Riferimenti normativi
+- Capitolato C6 - InnovaCity: Smart city monitoring platform:
+https://www.math.unipd.it/~tullio/IS-1/2023/Progetto/C6.pdf \
+https://www.math.unipd.it/~tullio/IS-1/2023/Progetto/C6p.pdf
+
+- Norme di Progetto
+
+- Regolamento progetto didattico:
+https://www.math.unipd.it/~tullio/IS-1/2023/Dispense/PD2.pdf
+
+=== Riferimenti informativi
+Analisi dei requisiti - corso di Ingegneria del Software a.a. 2023/2024: \
+https://www.math.unipd.it/~tullio/IS-1/2023/Dispense/T5.pdf \
+Cardin	Analisi e descrizione delle funzionalità: Use Case e relativi diagammi (UML) - corso di Ingegneria del Software a.a. 2023/2024: \
+https://www.math.unipd.it/~rcardin/swea/2022/Diagrammi%20di%20Attivit%C3%A0.pdf
+
+= Descrizione
+
+== Obiettivi del prodotto
+L'obiettivo del prodotto è la creazione di una piattaforma che funga da monitor ad un utente, che in questo caso è rappresentato da un amministratore pubblico di una #glossary("smart city"). L'utente sarà in grado di controllare, attraverso la consultazione di una dashboard, lo stato della città, sotto i punti di vista ambientali, logistici e di sicurezza. In questo cruscotto contterrà delle visualizzazioni grafiche basate sui dati, che per il progetto in questione saranno prodotti da una simulazione realistica.
+
+== Funzionalità del prodotto
+Il prodotto si compone di due parti principali:
+- *Un simulatore* che produce dei dati realistici, di svariate tipologie. Con realistici si intende verosimili in base al tipo di dato, ad es. un #glossary("sensore") della temperatura avrà un andamento di dati rilevati approssimabile ad una sinusoide.
+  - Questi dati che vengono raccolti e archiviati in un database #glossary("OLAP") e vengono passati ad un sistema di stream processing che permette di disaccoppiare lo stream di informazioni raccolto.
+- *Una dashboard* che permette di visualizzare i dati accumulati, sottoforma di grafici.
```

Comment:
che permetta* 
e aggiungerei che si possono effettuare query in linguaggio sql

Reply:
La prima correzione non la farei perché se nel primo punto ho scritto "un simulatore che produce", nel secondo punto sta bene "una dashboard che permette".
Per la seconda parte, non andrei specifico sulle tecnologie, come discusso in precedenza